docs/development.md

```
@@ -0,0 +1,127 @@
+# Development
+
+Working on `ovos-bus-client` itself, or hacking on a fork.
+
+## Repository layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)